server/routes.js

```diff
@@ -0,0 +1,12 @@
+
+const express = require('express');
+const { UserValidator } = require('./http/Validators/UserValidator');
```

Comment:
could we just have a validators directory, instead of giving a HTTP parent directory?

Reply:
Yeah, this will change when I make 2 PRs out of this. There will be only the `validators` directory.